components/Header.tsx

```diff
@@ -29,28 +30,24 @@ export default function Header() {
       </nav>
 
       {/* Mobile Menu */}
-      <nav className="sm:hidden w-full p-2">
-        <div className="grid grid-cols-5">
-          <div className="flex justify-start col-span-4">
-            <Link href="/">
-              <a className="text-2xl text-gray-400">Code Youngstown</a>
-            </Link>
-          </div>
-          <div className="flex justify-end col-span-1">
-            <button
-              className="text-gray-400 text-xl py-1 px-4 rounded-md"
-              onClick={ToggleMenu}
-            >
-              <FontAwesomeIcon icon={faBars} />
-            </button>
-          </div>
-        </div>
+      <nav className="sm:hidden flex flex-wrap justify-between items-center pt-1">
+        <Link href="/">
+          <a className="ml-2 text-2xl text-left text-gray-400">
+            Code Youngstown
+          </a>
+        </Link>
+        <button
+          className="text-gray-400 text-xl mr-4 flex text-right"
+          onClick={ToggleMenu}
+        >
+          <FontAwesomeIcon icon={faBars} />
+        </button>
 
-        <div id="mobile-nav-links" className={`w-full mt-2 ${menuLinks}`}>
+        <div className={`w-full mt-2 ${menuLinks}`}>
           <ul className="w-full">
             {Pages.map(([title, url]) => (
               <Link key={title} href={url}>
-                <li className="hover:underline hover:text-gray-300 border p-2">
+                <li className="hover:underline hover:text-gray-300 border p-1">
```

Comment:
Interestingly, using `hover:underline` here seems to be breaking having the pointer when you mouse over the mobile nav links